recipe/meta.yaml

```diff
@@ -1,29 +1,60 @@
-# Switch the version of this recipe based on the VC_VERSION environment variable.
-{% set VERSION = os.environ.get('VC_VERSION', '9')|string %}
-{% set runtime_years = {'9': '2008', '10': '2010', '14': '2015'} %}
-{% set runtime_year = runtime_years[VERSION] %}
+# version numbering is explained at https://stackoverflow.com/questions/42730478/version-numbers-for-visual-studio-2017-boost-and-cmake
+#     This version number follows the "platform toolset"
+{% set vcver="14" %}
+{% set vsyear="2015" %}
+{% set fullver="14.0.25420" %}
 
 package:
-  name: vc
-  version: {{ VERSION }}
+  name: vs{{ vsyear }}
+  version: {{ fullver }}
 
 build:
-  number: 0
-  track_features:
-    - vc{{ VERSION }}
-  skip: true  # [not win]
+  number: 11
```

Comment:
Why is this `11`?

Reply:
Because defaults is `10` and I intend to keep the recipes in sync.